packages/js/components/src/image-gallery/image-gallery-item.scss

```diff
@@ -0,0 +1,22 @@
+.woocommerce-image-gallery__item {
+    height: 146px;
+    width: 146px;
+    border: 1px solid $gray-200;
+    border-radius: 2px;
+    overflow: hidden;
+
+    img {
+        max-width: 100%;
+    }
+
+    .woocommerce-pill {
+        background: #007CBA;
```

Comment:
I wonder if we could use something like `background: var(--wp-admin-theme-color);` instead of fixing the color.

Reply:
Definitely!  Good call.